doc/refmodel.rst

```diff
@@ -36,8 +36,10 @@ Methods
 
 **General**
 
-* ``getNumVars(m::Model)`` - returns the number of variables associated with the ``Model m``.
-* ``getNumConstraints(m::Model)`` - returns the number of constraints associated with the ``Model m``.
+* ``MathProgBase.numvar(m::Model)`` - returns the number of variables associated with the ``Model m``.
```

Comment:
Good point, actually the re-export doesn't work because ``numvar`` are never imported into the JuMP namespace. Should we just not export them?